components/CollectiveCard.js

```diff
@@ -181,20 +181,15 @@ class CollectiveCard extends React.Component {
     const truncatedDescription = collective.description && firstSentence(collective.description, 80);
     const description = collective.description;
 
-    let route, params;
+    let route;
     if (collective.type === 'EVENT') {
-      route = 'event';
-      params = {
-        parentCollectiveSlug: collective.parentCollective?.slug || 'collective',
-        slug: collective.slug,
-      };
+      route = `${collective.parentCollective?.slug || 'collective'}/events/${collective.slug}`;
     } else {
-      route = 'collective';
-      params = { slug: collective.slug };
+      route = collective.slug;
```

Comment:
It's important to start the URLs with a `/`. See https://opencollective-frontend-bj8twodx4.vercel.app/babel/redeem, for example, the link is rendered as: https://opencollective-frontend-bj8twodx4.vercel.app/babel/babel

Reply:
@Betree : Are you sure? Because when I go to https://opencollective-frontend-bj8twodx4.vercel.app/babel/redeem and click on the link I get redirected to https://opencollective-frontend-bj8twodx4.vercel.app/babel.

![Peek 2021-02-15 06-48](https://user-images.githubusercontent.com/12435965/107960726-d0c0c980-6f59-11eb-9d93-096502728839.gif)

Reply:
The redirect indeed works, but there's definitely something going on with the `href`:

![image](https://user-images.githubusercontent.com/1556356/107978484-224e7000-6fbd-11eb-9d45-23d1ed85dd08.png)

Reply:
@Betree : You are absolutely right indeed. All the `next/link` examples points to the fact the route should be prefixed with the `/`. So I've went ahead and done this change. Seems now it's working correctly. 😄